Dockerfile

```diff
@@ -27,90 +29,127 @@
 FROM ubuntu:22.04 AS os
 
 # tzinfo hangs without this
-ARG DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND='noninteractive'
 
 # ---------------------------------------------------------------------
 # build
 # ---------------------------------------------------------------------
 
 FROM os AS build
 
+# limit cpus so don't run out of memory on local machine
+# symptom: get error - "c++: fatal error: Killed signal terminated program cc1plus"
+# can turn off if building in cloud
+ARG CONAN_CPU_COUNT=10
```

Comment:
We definitely need to change this to `2`. (I should have mark the PR as a draft :man_facepalming:.)